whois-commons/src/main/java/net/ripe/db/whois/common/domain/serials/SerialEntry.java

```diff
@@ -24,10 +25,20 @@ public SerialEntry(final Operation operation, final boolean atLast, final int ob
         rpslObject = RpslObject.parse(objectId, blob);
     }
 
+    public SerialEntry(final int serialId, final Operation operation, final boolean atLast, final int objectId, final byte[] blob) {
+        this(operation, atLast, objectId, 0, 0, blob);
+        rpslObject = RpslObject.parse(objectId, blob);
+        this.serialId = serialId;
+    }
+
     public static SerialEntry createSerialEntryWithoutTimestamps(final Operation operation, final boolean atLast, final int objectId, final byte[] blob){
         return new SerialEntry(operation, atLast, objectId, 0, 0, blob);
```

Comment:
Serial id here? Also I do not thunk we need timestamp in that case use this constructor?